app/resources/v1/country_resource.rb

```
@@ -13,7 +13,6 @@ class CountryResource < JSONAPI::Resource
     has_many :required_operator_documents
     has_many :required_gov_documents
     has_many :governments
-    has_many :monitors
```

Aren't we using this in the apps? (isn't there any request on the observations tool or the FE that does countries?include=monitors)?

---

There is no model `monitor` that never worked and errors were silenced by `do; end` blocks in routes

---

I have no idea why those empty blocks in `routes.rb` silenced many resource configuration errors, but I always wanted to remove those blocks as it looked just weird.